cfsdk/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
 
     defaultConfig {
 
-        def appVersionCode = 7
-        def appVersionName = "\"0.0.7\""
+        def appVersionCode = 8
```

Comment:
Same comment as java. Please use main branch for snapshots, only release branches should contain the final versions.